tldr/hybrid_extractor.py

```
@@ -342,6 +351,14 @@ def extract(self, file_path: str | Path, base_path: str | None = None) -> Module
                     return result
             logger.debug(f"Tree-sitter-elixir not available or failed, using Pygments for {suffix}")
 
+        # PHP - use tree-sitter-php if available
+        if suffix in self.PHP_EXTENSIONS:
+            if TREE_SITTER_PHP_AVAILABLE:
+                result = self._try_tree_sitter(self._extract_php, file_path, "php")
+                if result:
+                    return result
+            logger.debug(f"Tree-sitter-php not available or failed, using Pygments for {suffix}")
```

Ruff (0.15.7)

[warning] Logging statement uses f-string

(G004)